packages/compiler/testing/test-server-host.ts

```diff
@@ -32,6 +32,7 @@ export async function createTestServerHost(options?: TestHostOptions) {
 
   const serverHost: TestServerHost = {
     ...fileSystem,
+    throwInternalErrors: true,
```

Comment:
Is this used somewhere else?

Reply:
Yes, here: https://github.com/microsoft/cadl/pull/868/files#diff-bef5d28a42faec8254ad498a3faeb47aeb44d7d3e9f5fb12662b5a2d49d0e791R426

I wanted unit tests to not hide unexpected exceptions in a diagnostic.

Reply:
Seems github doesn't go to linked diff in "large diff".

This is where we use it in serverlib.ts
![image](https://user-images.githubusercontent.com/75470/186232674-06a8566a-613a-42a2-848f-4b54a5c3b803.png)